devito/parameters.py

```diff
@@ -147,6 +147,7 @@ def _signature_items(self):
     'DEVITO_OPT': 'opt',
     'DEVITO_MPI': 'mpi',
     'DEVITO_TOPOLOGY': 'topology',
+    'DEVITO_DEVICE_DEFAULT': 'deviceid',
```

Comment:
Why not just DEVICEID to match the config

Reply:
yes, changing